test/parallel/test-process-nice.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+if (common.isWindows || !common.isMainThread) {
+  assert.strictEqual(process.nice, undefined);
```

Comment:
This is no longer valid since we're now throwing.

However, I feel like we should just call `common.skip()` with an appropriate message to make it obvious that we're not 
 testing the real implementation.